edgedb-show-secrets.sh

```diff
@@ -82,7 +82,7 @@ edb_gs_show_secrets() (
 
   while IFS="=" read -r k v; do
     map["$k"]=$v
-  done < "/etc/edgedb.secrets"
+  done < "/tmp/edgedb.secrets"
```

Comment:
It seems like the dot was changed to a slash in `docker-entryphone-funcs.sh`. Am I reading that correctly? 

```suggestion
  done < "/tmp/edgedb/secrets"
```

Reply:
Yeah, it's a typo.